src/linear_algebra/affine_space/affine_subspace.lean

```diff
@@ -378,6 +378,23 @@ begin
     exact vsub_mem_direction hp hq2 }
 end
 
+instance to_add_torsor (s : affine_subspace k P) [nonempty s] : add_torsor s.direction s :=
+{ vadd := λ a b, ⟨(a:V) +ᵥ (b:P), vadd_mem_of_mem_direction a.2 b.2⟩,
+  zero_vadd' := by simp,
+  vadd_assoc' := λ a b c, by { ext, apply add_torsor.vadd_assoc' },
+  vsub := λ a b, ⟨(a:P) -ᵥ (b:P), (vsub_left_mem_direction_iff_mem a.2 _).mpr b.2 ⟩,
+  nonempty := by apply_instance,
+  vsub_vadd' := λ a b, by { ext, apply add_torsor.vsub_vadd' },
+  vadd_vsub' := λ a b, by { ext, apply add_torsor.vadd_vsub' } }
```

Comment:
The `TODO` in the module doc string should be removed now this instance is present.